docs/src/config.md

```diff
@@ -0,0 +1,60 @@
+# Configuration File
+
+From v0.4.2, JuliaFormatter offers [`.prettierrc` style](https://prettier.io/docs/en/configuration.html) configuration file support.
```

Comment:
i recently tagged v0.4.2 so this should be v0.4.3

Reply:
thanks for pointing this out ;)